apps/home/blocks.py

```diff
@@ -191,6 +191,6 @@ class TeaserBlockImageRight(blocks.StructBlock):
         required=False
     )
 
-    # class Meta:
-    #     template = 'apps_home/blocks/gruenbuch_faq_block.html'
-    #     icon = 'arrow-down'
+    class Meta:
+        template = 'apps_home/blocks/teaser_image_right.html'
+        icon = 'image'
```

Comment:
This isn't from this pull request and doesn't need to be refactored now but if we need to go over this code again we can actually have just 1 block called aligned image and do like is done [here](https://github.com/liqd/a4-opin/blob/0a759f5419b03b483b1549d0571436f437a0eafc/home/models/blocks.py#L96) in Opin

Reply:
Yes, thanks!!